vehicle/raw_vehicle_cmd_converter/include/raw_vehicle_cmd_converter/node.hpp

```diff
@@ -87,6 +87,9 @@ class RawVehicleCommandConverterNode : public rclcpp::Node
   AckermannControlCommand::ConstSharedPtr control_cmd_ptr_;
   AccelMap accel_map_;
   BrakeMap brake_map_;
+  SteerMap steer_map_;
+  // TODO(tanaka): consider accel/brake pid too
+  PIDController steer_pid_;
   bool ff_map_initialized_;
```

Comment:
Please remove this member variable. There is no situation running the converter without map. If the map load fails, throw an exception and kill the node.